app/screens/Reusable/Timers/TimerTitle/TimerTitle.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { SafeAreaView, View } from 'react-native';
+
+import { ExtendedText, Timer } from '../../../../components';
+import { ITimerTitleScreenProps } from './TimerTitle.types';
+import { styles } from './TimerTitle.styles';
+import { generalStyles } from '../../../../utils/styles';
+
+export const TimerTitleScreen: React.FC<ITimerTitleScreenProps> = ({
+  route,
+}) => {
+  const { t } = useTranslation();
+
+  const { duration, title } = route.params.data;
+
+  return (
+    <SafeAreaView style={generalStyles.flex}>
+      <View style={styles.container}>
+        <Timer duration={duration} isStart={true} />
+        {!!title && (
+          <ExtendedText preset="large-title" style={styles.title}>
+            {t(title)}
+          </ExtendedText>
```

Comment:
`titleKey` is more preferable here

Reply:
Anna said to use these names for props:
- title
- description
- tellMoreTitle
- tellMoreDescription
- backgroundImage
- tellMoreBackgroundImage
- description?